drivers/flash/spi_nor.c

```diff
@@ -348,11 +490,19 @@ static int spi_nor_configure(struct device *dev)
 	data->spi_cfg.cs = &data->cs_ctrl;
 #endif /* DT_INST_0_JEDEC_SPI_NOR_CS_GPIOS_CONTROLLER */
 
+	/* Might be in DPD if system restarted without power cycle. */
+	(void)exit_dpd(dev);
```

Comment:
If you add such (void), maybe you'll need it everywhere where relevant?

Reply:
Easier to remove it than try to explain why it makes more sense here than other places, so it's gone.  Thanks for the approval.

Reply:
ok, I was also wondering why not just removing it. We do not enforce returned code check everywhere after all

Reply:
OK, so the reason I did it is that here I really don't care if it fails.  We've already confirmed that we can communicate with the device, and we only need to do this to handle a case where a soft reset with DPD active switched to code that doesn't enable DPD when idle.

The draft Zephyr coding standard does require checking return values everywhere (though the estimate of 186 violations in the code base seems ridiculously low) so all this may have to change anyway.  (Depending on what functions are deemed to "return error information".)